comitup/mdns.py

```diff
@@ -129,6 +138,34 @@ def get_interface_mapping():
             pass
 
     return mapping
+    
+
+def load_data():
```

Comment:
I'm not OK with the duplication of this function. Can you either extend the init_(mgr)() tree starting in comitup.py, or move load_data to config.py, or a new file?

Reply:
Fixed with 9969ab2.